gitbook-docs/request_response.md

```diff
@@ -0,0 +1,130 @@
+# Request/Response
+
+## WebSockets and similar transports
```

Comment:
`similar transports` is ambiguous: WS as an extension to HTTP is much more similar to WS than it is to it's distant forefather, TCP. I thinks we should define this such as:

`WebSockets and other full-duplex streaming message-based protocols; e.g. TCP streams, UDP, 1-wire, etc`

Reply:
My English sentence could use some TLC

Reply:
How about `WebSockets and other full-duplex protocols`

Reply:
Works for me